sample-scripts/call-recording/Call_Recording.php

```diff
@@ -0,0 +1,121 @@
+<?php
+// base urls
+$base_url = "https://public-api.sonetel.com/";
+
+// function will change the format of date
+function date_format_change($date){
+    return str_replace(":", "%3A", $date);
+}
+
+//Enter Date
+$created_date_max = "ENTER THE MAX DATE";
```

Comment:
Perhaps it is better to use the date format we expect. For example, `$created_date_max = "yyyy-mm-ddThh:mm:ssZ"`.

This makes it clear to the customer what date format we're expecting.

Reply:
@aashish-joshi Resolved